src/Views/FirmwareView.vala

```diff
@@ -120,12 +120,12 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
                 add_device (devices[i]);
             }
 
-            placeholder_alert_view.title = _("Firmware Updates Are Not Available");
-            placeholder_alert_view.description = _("Firmware updates are not supported on this or any connected devices.");
+            placeholder_alert_view.title = _("No new firmware available");
+            placeholder_alert_view.description = _("None of your devices support installing new firmware this way.");
             update_list.show_all ();
         } catch (Error e) {
-            placeholder_alert_view.title = _("The Firmware Service Is Not Available");
-            placeholder_alert_view.description = _("Please make sure “fwupd” is installed and enabled.");
+            placeholder_alert_view.title = _("Firmware service unavailable");
```

Comment:
I don't have hugely strong feelings on the use of "unavailable" versus "not available", but I think it sounds more natural to say "The firmware service is unavailable" than to exclude these words

Reply:
I think the minimal amount of text is what I want when trying to keep on top of what is very stressful and possibly non-reversible. Title case for a sentence looks off.

Reply:
```suggestion
            placeholder_alert_view.title = _("Could not connect to firmware service");
```